tests/automated_validation/test_interface.py

```diff
@@ -169,6 +169,36 @@ def test_get_frame(sim_result_dir: Path) -> None:
     assert set(data2.columns) == {"test_rate", "reference_rate", "percent_error"}
 
 
+def test_metadata(sim_result_dir: Path, mocker: MockFixture) -> None:
+    """Ensure that we can summarize a comparison"""
+    measure_key = "cause.disease.incidence_rate"
+    context = ValidationContext(sim_result_dir)
+    context.add_comparison(measure_key, "sim", "artifact")
+
+    mocker.patch(
```

Comment:
This is the best way I could think of testing that these methods work. I have to patch each of these but kept the mocked return values in a format we would expect.